app/routes.js

```diff
@@ -51,15 +51,23 @@ router.get('*/check', (req, res, next) => {
   return isDapAvailable(req, res, next);
 });
 
+// router.get('*/start', (req, res, next) => {
+//   req.session.data.skipStartPageValidation = skipStartPageValidation;
+//   next();
+// });
+
 // If a start page in the version exists,
 router.get('*/eligibility', (req, res, next) => {
   const url = req.params[0];
   const startExists = fs.existsSync('app/views' + url + '/start.html')
 
   if (startExists) {
     const data = req.session.data;
+
```

Comment:
if we've changed the action  to itself, how are we ever getting to this page? how is this handler being hit?

Reply:
Ah man, I've totally misread. I thought the last file changed was from the select-eligibility-cards page, not the launch prototype button. Sorry totally missed that

Reply:
Not a problem!